developers/documentation/docs_template.md

```diff
@@ -0,0 +1,146 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Docs template
+
+This template will guide you to write a well formatted document and prepare it for contribution to napari.org.
+
+## Prerequisites
+
+Fill out this section with a list of things the reader will need to prepare to be able to follow this document.
+Include things like:
+- links to any existing napari.org tutorials or how-to guides that can help the user fulfill these prerequisites
+- the level of python/napari knowledge required to follow this document
+    - try to be specific about what skills are needed e.g. 
+    'connecting callbacks to layer events' or 'using matplotlib to produce plots'
+- plugins that should be installed
+- python packages that should be installed (don't list napari or its dependencies)
+- links to sample data that can be used to follow your document
+  - **don't add this data to the repository**
+
+## Write your document
+Fill out the main content of the document - your explanation, how-to steps or tutorial. 
+
+## Include pictures
```

Comment:
based on https://github.com/napari/napari/pull/3815, we should also make it clear _where_ such images should be stored (probably not in the repo, especially if we move forward with `napari/napari` as the location of the docs)

Reply:
yeah, we might want to put this in extra-highlights, and even add to our core developer guide to look out for this in review as the negative consequences of this can be quite large (having to remove commit from GitHub history). Thanks for flagging @neuromusic!!